app/controllers/questions/stimulus_recommendation_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Questions
```

Comment:
So this is perhaps being a bit lazy since this really does not use much of the QuestionsController since there really isn't a question or form on the page. I also considered making this an action in the PublicPagesController but that did not seem quite right either. It could be just its own stand alone controller inheriting from application controller using the default layout.

I did think that perhaps some special tracking might be desired at some point to see if clients are going to the FAQ page from here in which case it might make sense to make the button a form. Of course that could be easily tracked in other ways as well.

Reply:
we talked about it and decided to put it in public pages because that's the pattern we use for the eligibility offboarding path